[DNMY] New NUT configurator feature image

boricj opened the pull request:
Part of the feature image "new-nut-configurator".

jean-baptisteboric-eaton committed:
Database integration with proxy objects

Signed-off-by: Jean-Baptiste Boric <Jean-BaptisteBORIC@Eaton.com>

ericclappier-eaton committed:
Remove fty-common-nut dependencie

etnKlendathu committed:
Fixed clang compilation

jean-baptisteboric-eaton committed:
Regenerate zproject

Signed-off-by: Jean-Baptiste Boric <Jean-BaptisteBORIC@Eaton.com>

stark-dev reviewed (approved):
not enough skilled on the project to 'approve', but it seems ok to me ;)

jimklimov committed:
Problem: consumer fty-certificate-generator dies upon quick restart

Solution: declare that this service is up only when the socket is activated,
only then the consumers with proper Requires+After can begin to start.

Signed-off-by: Jim Klimov <EvgenyKlimov@eaton.com>

jimklimov committed:
configure.ac : add acinclude.m4 to define @socketSecurityWallet(Dir)@

jimklimov committed:
fty-security-wallet.service.in fty-security-wallet.conf.in fty-security-wallet.cfg.in : pass @socketSecurityWallet(Dir)@ via autoconf

jimklimov committed:
fty-security-wallet.cfg.in : end with EOL

jimklimov committed:
fty-security-wallet.service.in : allow a generous (for slower hosts) sleep after startup to "ensure" malamute listener appears

jimklimov committed:
Move ipc-meta-setup scriptlet from src/ to setup/

jimklimov committed:
fty-security-wallet.service.in : typo fix, broke escaped shell syntax

arnaudquette-eaton committed:
Use systemd notification to announce service/socket readiness

Signed-off-by: Arnaud Quette <ArnaudQuette@Eaton.com>

jimklimov committed:
project.xml: whitespace fixes

jimklimov committed:
Mark libsystemd-dev as optional dependency, and travis-check compilability with it disabled

jimklimov committed:
fty_security_wallet.cc: fix whitespace

jimklimov committed:
fty_security_wallet.cc : do not include sd-daemon.h here, we have it via project standard include files

jimklimov committed:
install.xml : comment for config delivered right to /etc

jimklimov committed:
fty-security-wallet.cc : there is no secw_server(/verbose) in fty-security-wallet.cfg.in

jimklimov committed:
fty-security-wallet.service.in : change to Type=notify for sd_notify() support

jimklimov committed:
fty_security_wallet.cc : log_debug the sd_notify calls

jimklimov committed:
fty_security_wallet.cc : sd_notify that we are ready to serve AFTER starting the listeners (or at least, telling them to start)

jimklimov committed:
acinclude.m4 : offload SECW socket path arg processing into routines that are cleaner to sync and call

jimklimov committed:
acinclude.m4 : offload optional AX_SET_SYSTEMD_SERVICE_TYPE processing into a routine that is easier to disseminate

jimklimov committed:
acinclude.m4 : comment that src/fty-security-wallet.conf is generated here unlike most other components, and only do this if configured WITH_SYSTEMD_UNITS

jimklimov committed:
fty-security-wallet.service.in : extend the test for listened socked like done in malamute.service

jimklimov committed:
Bump copyrights to 2020

jimklimov committed:
.gitignore : resync with master and other 42ity component patterns

jimklimov committed:
.travis.yml : Mark libsystemd-dev as optional dependency, and travis-check compilability with it disabled

jimklimov committed:
src/.gitignore : comment the manual edition

jimklimov committed:
fty-security-wallet.service.in : make it non-fatal when bios-db-rw does not exist, and comment why

jimklimov committed:
.gitignore : update distcheck products ignorance pattern

boricj commented:
Feature image "new-nut-configurator" is obsolete.
